app/dashboard/views.py

```diff
@@ -1620,6 +1658,24 @@ def profile(request, handle):
     context['sent_kudos_count'] = sent_kudos.count()
     context['verification'] = profile.get_my_verified_check
 
+    unrated_funded_bounties = Bounty.objects.prefetch_related('fulfillments', 'interested', 'interested__profile') \
+        .filter(
+        bounty_owner_github_username=profile.handle,
+        idx_status='done')
+
+    unrated_contributed_bounties = Bounty.objects.current().filter(interested__profile=profile).filter(interested__status='okay') \
+        .filter(interested__pending=False).filter(idx_status='done')
+
+    context['unrated_funded_bounties'] = []
+    context['unrated_contributed_bounties'] = []
+    for bounty in unrated_funded_bounties:
+        if not FeedbackEntry.objects.filter(bounty=bounty, feedbackType='approver'):
```

Comment:
I think this process of getting all the bounties, then checking each bounty to see if feedback exists is going to perform slowly. We should find a way to just get them in one query for each context entry. Maybe something like: `context['unrated_funded_bounties'] = Bounty.objects.prefetch_related(...,).filter(...,feedbacks__isnull=)` using the `related_name` property of `FeedbackEntry` here: https://github.com/gitcoinco/web/blob/0d76622b37c87da737d2a01aaca11c3f3edfdb5c/app/dashboard/models.py#L3010